docs/authorization/scopes_and_consents/index.rst

```diff
@@ -0,0 +1,18 @@
+Scopes and Consents
+===================
+
+Globus uses OAuth2 scopes to control access to different APIs and allow
+applications provide a least-privilege security guarantee to their users.
+
+Within the Globus SDK, Scopes and Consents are modeled using several objects
+which make learning about and manipulating these data easier.
+
+For more information, see
+`this docs.globus.org explanation of scopes and consents
+<https://docs.globus.org/guides/overviews/clients-scopes-and-consents/>`_.
+
+.. toctree::
+    :maxdepth: 1
+
+    scopes
+    mutable_scopes
```

Comment:
Move this to the top of the page (or signal in a more apparent way that this is a nav link structure).

Otherwise it looks at a glance to me like "these docs have been moved to docs.globus.org/guides/overview/clients_scopes_and_consents."

Reply:
Yeah, I'll rearrange. I don't know exactly how yet; let me get in there and see what looks right.

Reply:
I broke things apart into sections, so that the overview/background on scopes with the docs.globus.org link could go in a separate section. Take a look and LMK if we can mark this resolved!